etc/grafana/dashboards/overview.json

```diff
@@ -9788,1235 +9788,61 @@
-          "legendFormat": "{{quantile}} percentile",
-          "range": true,
-          "refId": "A"
-        },
-        {
-          "datasource": {
-            "type": "prometheus",
-            "uid": "${DS_PROMETHEUS}"
-          },
-          "editorMode": "code",
-          "expr": "reth_blockchain_tree_make_canonical_duration{action=\"insert old canonical chain\", quantile=\"0.9\"} ",
-          "hide": false,
-          "instant": false,
-          "legendFormat": "{{quantile}} percentile",
-          "range": true,
-          "refId": "B"
-        },
-        {
-          "datasource": {
-            "type": "prometheus",
-            "uid": "${DS_PROMETHEUS}"
-          },
-          "editorMode": "code",
-          "expr": "reth_blockchain_tree_make_canonical_duration{action=\"insert old canonical chain\", quantile=\"0.95\"} ",
-          "hide": false,
-          "instant": false,
-          "legendFormat": "{{quantile}} percentile",
-          "range": true,
-          "refId": "C"
-        },
-        {
-          "datasource": {
-            "type": "prometheus",
-            "uid": "${DS_PROMETHEUS}"
-          },
-          "editorMode": "code",
-          "expr": "reth_blockchain_tree_make_canonical_duration{action=\"insert old canonical chain\", quantile=\"0.99\"} ",
-          "hide": false,
-          "instant": false,
-          "legendFormat": "{{quantile}} percentile",
-          "range": true,
-          "refId": "D"
-        },
-        {
-          "datasource": {
-            "type": "prometheus",
-            "uid": "${DS_PROMETHEUS}"
-          },
-          "editorMode": "code",
-          "exemplar": false,
-          "expr": "reth_blockchain_tree_make_canonical_duration{action=\"insert old canonical chain\", quantile=\"0.999\"} ",
-          "hide": false,
-          "instant": false,
-          "legendFormat": "{{quantile}} percentile",
-          "range": true,
-          "refId": "E"
-        }
-      ],
-      "title": "Insert Old Canonical Chain",
-      "type": "timeseries"
-    }
-  ],
-  "refresh": "30s",
-  "revision": 1,
-  "schemaVersion": 38,
-  "style": "dark",
-  "tags": [],
-  "templating": {
-    "list": [
-      {
-        "current": {},
-        "datasource": {
-          "type": "prometheus",
-          "uid": "${DS_PROMETHEUS}"
-        },
-        "definition": "query_result(up)",
-        "hide": 0,
-        "includeAll": false,
-        "multi": false,
-        "name": "instance",
-        "options": [],
-        "query": {
-          "query": "query_result(up)",
-          "refId": "PrometheusVariableQueryEditor-VariableQuery"
+      {
+        "current": {},
+        "hide": 0,
+        "includeAll": false,
+        "label": "Blockchain Tree Canonicalization Duration Action",
+        "multi": false,
+        "name": "blockchain_tree_action",
+        "options": [],
+        "query": {
+          "query": "reth_blockchain_tree_make_canonical_duration{action=~\".*\"}",
```

Comment:
no need to use regex here, just grabbing the label values should work

```json
{
  "current": {},
  "datasource": {
    "type": "prometheus",
    "uid": "${DS_PROMETHEUS}"
  },
  "definition": "label_values(reth_blockchain_tree_make_canonical_duration{instance=\"$instance\"},action)",
  "hide": 0,
  "includeAll": false,
  "label": "Blockchain Tree Canonicalization Action",
  "multi": false,
  "name": "blockchain_tree_action",
  "options": [],
  "query": {
    "query": "label_values(reth_blockchain_tree_make_canonical_duration{instance=\"$instance\"},action)",
    "refId": "PrometheusVariableQueryEditor-VariableQuery"
  },
  "refresh": 1,
  "regex": "",
  "skipUrlSync": false,
  "sort": 0,
  "type": "query"
}
```

<img width="576" alt="image" src="https://github.com/paradigmxyz/reth/assets/5773434/b5d378f0-18ff-4eb0-900f-f19f0da6ba49">